python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/logging.schema.yml

```diff
@@ -128,6 +132,21 @@ keys:
       event:
         type: dict
         keys:
+          congestion_drops:
```

Comment:
we can create a single key - `congestion_drops_interval` instead of a dict

Reply:
I don't have an opinion, just trying to better understand so I can make cleaner PRs in the future. Is there a standard that drives when to use nested keys vs a single key? As an example in the same file there's this structure:

https://github.com/aristanetworks/avd/blob/997453813a4fe391464636e632c48fdf81b609d7/python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/logging.schema.yml#L45-L53

- which is needlessly a dictionary. I assumed this was done to match the structure of the EOS CLI command and allow for future expansion of that command. I don't mind merging these two suggestions just trying to understand the standards so I can make sure I comply going forward.

Reply:
since there is only one command under congestion_drops, I think it can be combined to `congestion_drops_interval`

Reply:
@nathanmusser so essentially we are trying to limit the nested-ness of things these days. There may be things from before (maybe not so long sometimes things slip) that have more levels than necessary but we don't go through the deprecation process and live with it. But as a rule of thumb if there is only one key - we can have dict.